include/NovelRT/Ecs/ComponentCache.h

```diff
@@ -1,92 +1,95 @@
-// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
 
 #ifndef NOVELRT_ECS_COMPONENTCACHE_H
 #define NOVELRT_ECS_COMPONENTCACHE_H
 
-#include <unordered_map>
-#include <functional>
-#include <vector>
+#include "../Exceptions/OutOfMemoryException.h"
+#include "../Utilities/Event.h"
 #include "ComponentBuffer.h"
 #include "EcsUtils.h"
-#include "../Utilities/Event.h"
-#include "../Exceptions/OutOfMemoryException.h"
+#include <functional>
+#include <unordered_map>
+#include <vector>
 
 namespace NovelRT::Ecs
 {
     /**
-     * @brief Stores all ComponentBuffer instances currently initialised for this instance of the ECS. You should not be instantiating this yourself in a regular setup.
-     * 
+     * @brief Stores all ComponentBuffer instances currently initialised for this instance of the ECS. You should not be
+     * instantiating this yourself in a regular setup.
+     *
      */
     class ComponentCache
     {
-        private:
+      private:
```

Comment:
now THIS is interesting - I would like additional input on this indentation specifically. Do we want to keep this behaviour?

Reply:
I'm fine with it being dedented one, to be even with the brace for the class

Reply:
Resolved - I've dedented access modifiers to be even with the bracing for the class.